### PR TITLE
Remove attr- from HTML link fragments

### PR DESCRIPTION
**Pull request by OnkarRuikar**
We've removed HTMLAttrDef macro. The PR updates `#attr-*` fragments which were/are broken.

SVG fragments had been broken for a long time. May be because the author thought SVGAttrDef works like HTMLAttrDef macro.

### CODE REVIEW

File: files/en-us/web/html/element/link/index.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
     If the attribute is not present, the resource is fetched without a {{Glossary("CORS")}} request (i.e. without sending the `Origin` HTTP header), preventing its non-tainted usage. If invalid, it is handled as if the enumerated keyword **anonymous** was used.
     See [CORS settings attributes](/en-US/docs/Web/HTML/Attributes/crossorigin) for additional information.
 
-- `disabled` {{Deprecated_Inline}} {{Non-standard_Inline}}
+- `disabled` {{Non-standard_Inline}}
```
Comment (teoli2003) [inline files/en-us/web/html/element/link/index.md:180]: I don't think this should be in this PR.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/html/element/link/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/html/element/link/index.md:180]: I made a blunder in last BCD sync(which was run manually after long time due to `tags` removal): yari hadn't picked up latest BCD and seeing it still deprecated in BCD on MDN I reverted the script change.\
The change remained in git stash and got picked up with this PR.

Do you want me to create a separate PR for this? or wait for next BCD sync where it'll be automatically handled?

Comment (teoli2003) [reply files/en-us/web/html/element/link/index.md:180]: Just wait for next BCD sync.

Comment (OnkarRuikar) [reply files/en-us/web/html/element/link/index.md:180]:
```suggestion
- `disabled` {{Deprecated_Inline}} {{Non-standard_Inline}}
```

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**